File: mario.py
from cs50 import get_int
t = get_int("Height :")
while t <= 0 or t > 8:
    t = get_int("Height:")
for p in range(1, t + 1, 1):
    s = t - p
    for _ in range(s):
        print(" ", end="")
    for _ in range(p):
        print("#", end="")
    print("  ", end="")
    for _ in range(p):
        print("#", end="")
    print()

File: mario.c
#include <cs50.h>
#include <stdio.h>

void row(int space, int i);

int main(void)
{
    int number, space, i, j;
    do
    {
        number = get_int("Height: ");
    }
    while (number < 1 || number > 8);
    for (i = 0; i < number; i++)
    {
        space = number - 1 - i;
        row(space, i + 1);
    }
}
void row(int space, int i)
{
    int k;
    for (k = 0; k < space; k++)
    {
        printf(" ");
    }
    for (k = 0; k < i; k++)
    {
        printf("#");
    }
    printf("  ");
    for (k = 0; k < i; k++)
    {
        printf("#");
    }
    printf("\n");
}


